mm2src/mm2_main/src/lp_ordermatch/my_orders_storage.rs

```diff
@@ -724,6 +724,7 @@ mod tests {
             base_orderbook_ticker: None,
             rel_orderbook_ticker: None,
             p2p_privkey: None,
+            swap_version: 1,
```

Comment:
I think should be a named const instead of '1'

Reply:
Done https://github.com/KomodoPlatform/komodo-defi-framework/pull/2261/commits/9ec0babe1802613ce77967a71051d82cde4c6caa